bad_code.java

```diff
@@ -0,0 +1,54 @@
+public class BadCodeExample {
```

Comment:
Use a more meaningful name for the `BadCodeExample` class, as it doesn't provide any information about its purpose [medium]